auth/src/index.ts

```diff
@@ -0,0 +1,12 @@
+export interface Email {
+  value: string;
+  type?: string;
+}
+
+export interface Profile {
+  provider: string;
```

Comment:
Looking at code and experimenting with it live, it looks like provider field is optional - see the profile [sources](https://github.com/auth0/passport-auth0/blob/master/lib/Profile.js#L4). Should we fill those or leave unchanged?

Reply:
Let's leave it like this for now.